### ISSUE TITLE
feature: Support for custom icons

### ISSUE THREAD

**Issue by 4lch4** [fabiospampinato/vscode-projects-plus#71]
I've been using Projects+ for a while now and only _just_ learned of the ability to use icons for groups. From what I can tell there are two sources to choose from:

- [Octicons][0]
- ASCII Icons

I'm wondering if anyone would be interested in support for custom icons.

## Potential "Design"

My initial thought is there could be a new setting added that's called something like... `projects.iconsDir` and if it's present, check there for any icon files that match the given icon name. If not, then fall-back on whatever source mentioned above is enabled. That way you can still use Octicons/ASCII Icons but also have the option to use a custom icon.

## Conclusion

I'd love to be the one to try and make this feature a reality, so long as @fabiospampinato is alright with me putting together a PR. That said, thank you again Fabio for such an amazing extension.

[0]: https://iconify.design/icon-sets/octicon/

**Comment by fabiospampinato** [comment]
I don't think this is something that's supported by vscode, without their supported this extension can't just implement this.
